ibis/backends/base/sql/alchemy/__init__.py

```diff
@@ -752,7 +752,10 @@ def _compile_pyarrow_udf(self, udf_node: ops.ScalarUDF) -> str:
     def _register_udfs(self, expr: ir.Expr) -> None:
         with self.begin() as con:
             for sql in self._compile_udfs(expr):
-                con.exec_driver_sql(sql)
+                if callable(sql):
+                    sql(con)
```

Comment:
Maybe we can name this something other than `sql`?  `sql(con)` looks very strange.

Reply:
Changed to `register_udf`

Reply:
Ultimately went with a slightly different approach here, so the naming should be even clearer!